tools/ddrgen/src/scanners/dwarf/AixDwarfParser.hpp

```diff
@@ -19,6 +19,19 @@
 #include "config.hpp"
 #include "DwarfFunctions.hpp"
 
+/* PTR_SIZE varies for 32bit, 64bit */
+const int PTR_SIZE = 8;
+
+const string BUILT_IN_TYPES [36] = {"", "int", "char", "short", "long", "unsigned char",
+"signed char", "unsigned short", "unsigned int", "unsigned", "unsigned long", "void", 
+"float", "double", "long double", "integer", "boolean", "short real", "real", "stringptr",
+"character", "logical*1", "logical*2", "logical*4", "logical", "complex", "complex", "integer*1",
+"integer*2", "integer*4", "wchar", "long long", "unsigned long long", "logical*8", "integer*8",
```

Comment:
What is an integer*1, integer*2, logical*1, complex, etc? Are the previous base types insufficient to cover all cases? ie char for size 1, short for size 2, int for size 4, etc.

If the DwarfScanner and superset generator produce base types of "logical" and "integer", how would one translate these when generating the java code from the superset? The set of lowest base types should probably only have signed/unsigned versions for each size. For other types, I'd suggest creating a Die for a typedef with the name and setting its type to the corresponding base type of the appropriate size.